mtt/config/Run2/config_2017.py

```diff
@@ -373,8 +376,140 @@
         dataset.add_tag({"is_e_data", "is_egamma_data"})
     if dataset.name.startswith("data_pho"):
         dataset.add_tag({"is_pho_data", "is_egamma_data"})
-    if dataset.name.startswith("data_pho"):
-        dataset.add_tag({"is_pho_data", "is_egamma_data"})
+
+#
+# tagger configuration
+# (b/top taggers)
+#
+
+# 2017 b-tagging working points
+# https://twiki.cern.ch/twiki/bin/view/CMS/BtagRecommendation106XUL17?rev=15
+config_2017.x.btag_wp = {
+    "deepjet": {
+        "loose": 0.0532,
+        "medium": 0.3040,
+        "tight": 0.7476,
+    },
+    "deepcsv": {
+        "loose": 0.1355,
+        "medium": 0.4506,
+        "tight": 0.7738,
+    },
+}
+
+# 2017 top tagging working points (DeepAK8, 1% mistagging rate, )
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/DeepAK8Tagging2018WPsSFs?rev=4
+config_2017.x.toptag_wp = {
+    "deepak8": {
+        # regular tagger
+        "top": 0.344,
+        "w": 0.739,
+        # mass-decorrelated tagger
+        "top_md": 0.725,
+        "w_md": 0.925,
+    },
```

Comment:
I just checked the TWiki and it seems that the regular and mass-decorrelated tagger values are switched around. So the correct config should probably read:
```suggestion
        # regular tagger
        "top": 0.725,
        "w": 0.925,
        # mass-decorrelated tagger
        "top_md": 0.344,
        "w_md": 0.739,
    },
```

@jomatthi Could you double-check and adapt this if needed?

Reply:
Good catch, thank you!